docs/en/setup/backend/backend-pulsar-monitoring.md

```diff
@@ -0,0 +1,67 @@
+# Pulsar monitoring
+
+SkyWalking leverages Prometheus to collect metrics data from the Pulsar and leverages OpenTelemetry Collector to transfer the metrics to
```

Comment:
```suggestion
SkyWalking leverages OpenTelemetry Collector to collect metrics data in Prometheus format from the Pulsar and transfer the metrics to
```

Please update two docs about this. We don't leverage Prometheus. We only use the Prometheus endpoint of Pulsar and BookKeeper only.